### PR TITLE
Create 5-02-03.csv

### PR DESCRIPTION
**Pull request by anthoninlize**
Create 5-02-03.csv for initial Measurement/observing method (ocean)

### CODE REVIEW

File: tables_en/5-02-03.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+,ocean,"ocean,pressure",Piezoresistivity,Measurement of the change in electrical resistance of a material under mechanical stress.
+,ocean,"ocean,waves,direction,period,height",HF echo doppler spectra analysis method (HR radar),Remote sensing of surface currents employing radars frequencies
+,ocean,"ocean,current,direction,speed",GPS fixes sequence comparison,Measurement derived from the difference between two successives GPS positions
+,ocean,"ocean,dissolved oxygen",Clark electrode polarographic method,The dissolved oxygen concentration by counting the number of oxygen molecules per second
+,,,,(flux) that diffuse through the membrane from the ocean environment to the working electrode
+,ocean,"ocean,dissolved oxygen",Oxygen luminescence quenching method,The fluorescent indicator is a special complex embedded in a gas permeable sensing foil that is exposed to the surrounding water. The sensing foil is excited by modulated light; the sensor measures the phase of the returned light.
+,ocean,"ocean,pH",Potentiometric method,ion-sensitive field-effect transistor that responds to proton activity and
+,,,,a Ag/AgCl reference electrode
```
Comment (amilan17) [inline tables_en/5-02-03.csv:15]: @anthoninlize Is "a Ag/AgCl reference electrode" supposed to be a part of the definition for "Potentiometric"? Right now it appears in a new row preceded by empty cells.

File: tables_en/5-02-03.csv [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+,"atmosphere,ocean","atmosphere,ocean,temperature",Electrical resistivity,Measurement derived from the measurement of electrical resistivity
+,ocean,"ocean,salinity",Electrical conductivity,Measurement derived from the measurement of electrical conducivity
+,ocean,"ocean,salinity",Inductivity,Measurement derived from the measurement of magnetic inductance
+,ocean,"ocean,pressure",Piezoresistivity,Measurement of the change in electrical resistance of a material under mechanical stress.
+,ocean,"ocean,waves,direction,period,height",HF echo doppler spectra analysis method (HR radar),Remote sensing of surface currents employing radars frequencies
+,ocean,"ocean,current,direction,speed",GPS fixes sequence comparison,Measurement derived from the difference between two successives GPS positions
+,ocean,"ocean,dissolved oxygen",Clark electrode polarographic method,The dissolved oxygen concentration by counting the number of oxygen molecules per second
+,,,,(flux) that diffuse through the membrane from the ocean environment to the working electrode
```
Comment (amilan17) [inline tables_en/5-02-03.csv:12]: Is this part of the definition for "Clark electrode polarographic method"?

**Review by amilan17 — CHANGES_REQUESTED** [review]
@anthoninlize Thank you for creating this table. It looks like there are some errors in the CSV. May I also ask that you make the sentences complete (with a period at the end). Thanks.

File: tables_en/5-02-03.csv (hunk 1 above)
Comment (anthoninlize) [reply tables_en/5-02-03.csv:15]: Ahhh, indeed. I can update. What's the best way to handle this there? Closing this pull request and create another one with the fix?

**Comment by anthoninlize** [comment]
Sorry for that. I updated the CSV and re-requested a review. Thanks

**Review by amilan17 — APPROVED** [review]
Thank you for updating. It looks good now. You can add notations as well.

**Comment by anthoninlize** [comment]
Should notations be numbers or based on the name, following https://github.com/wmo-im/wmds/wiki/Guidance-for-spelling-of-notation?
If number, I assign them purely incrementally?

**Review by amilan17 — APPROVED** [review]

**Pull request merged**